### PR TITLE
Port to jbuilder

### PR DESCRIPTION
**Pull request by mseri**
The first plan was to make a proper breaking rename to: `xapi-rrd`, `xapi-rrd-unix`.

It turns out that such change would break a long list of dependencies and having all the fixes to the repositories and the spec files at the same time is too time consuming and error-prone.

I took the ported library and made it retrocompatible as we have done with some other packages.
Once this gets reviewed and merged, it requires a new release and the addition of the new `rrd` package to `xs-opam/packages/xs` (and update of `xapi-rrd`).

The plan is to address all the renaming together for all the libraries sometime in the (distant?) future.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,43 +1,24 @@
...
+uninstall:
+	jbuilder uninstall
 
-uninstall: setup.data
-	$(SETUP) -uninstall $(UNINSTALLFLAGS)
-	ocamlfind remove rrd || /bin/true
-
-reinstall: setup.data
-	$(SETUP) -reinstall $(REINSTALLFLAGS)
+tests:
```
Comment (gaborigloi) [inline Makefile]: I think this is usually singular.

Comment (mseri) [reply Makefile]: I keep jumping between singular and plural :D will never get it right

**Comment by mseri** [comment]
Travis failure is due to the usual `opam-ci` issue. I hope we can find a solution soon

**Comment by gaborigloi** [comment]
We'll need to release this and update xs-opam.

**Comment by mseri** [comment]
Ok, now at least the EXTRA_REMOTES test is green.

**Pull request merged**